sql/postgres/inspect.go

```diff
@@ -772,6 +772,14 @@ type (
 		Columns []*schema.Column
 	}
 
+	// Concurrently describes the CONCURRENTLY clause to instruct Postgres
+	// to build or drop the index concurrently without blocking the current table.
+	// https://www.postgresql.org/docs/current/sql-createindex.html#SQL-CREATEINDEX-CONCURRENTLY
+	Concurrently struct {
+		schema.Attr
+		Concurrently bool
+	}
```

Comment:
```suggestion
	Concurrently struct {
		schema.Attr
	}
```
The flag there is not needed. If it exists in the attached attributes, it's enough. 
See `NoInherit` as an example.